cockatrice/src/chatview.cpp

```diff
@@ -199,7 +199,7 @@ void ChatView::appendMessage(QString message, QString sender, UserLevelFlags use
             }
             // another user has been mentioned
             else {
-                int mentionEndIndex = message.indexOf(" ");
+                int mentionEndIndex = message.indexOf(QRegExp("\\W"), 1);// from 1 as @ is non-char
```

Comment:
Why not use `\b` for word boundaries? I've seen a user with a username starting with `#`